config/build-config.yaml

```diff
@@ -5,8 +5,7 @@ recipes:
   - name : tensorflow-base
     path : recipe
     resources:
-       memory: 200Gi  #[ppc64le]
```

Comment:
Our ppc cluster no more works when 200GB of RAM is requested. Hence keeping 10GB like we have for x86 cluster.